packages/client/modules/teamDashboard/components/ProviderRow/SlackNotificationRow.tsx

```diff
@@ -25,7 +25,8 @@ const labelLookup = {
   meetingStart: 'Meeting Start',
   MEETING_STAGE_TIME_LIMIT_END: `Meeting ${MeetingLabels.TIME_LIMIT} Ended`,
   MEETING_STAGE_TIME_LIMIT_START: `Meeting ${MeetingLabels.TIME_LIMIT} Started`,
-  TOPIC_SHARED: `Topic Shared`
+  TOPIC_SHARED: `Topic Shared`,
+  STANDUP_RESPONSE_SUBMITTED: 'Standup Response Submitted'
 } as Record<SlackNotificationEventEnum, string>
```

Comment:
+1 not related but it'd be nice to improve types along the way and change that to specifying a type instead of casting, it'll actually be typesafe then
```
const labelLookup: Record<SlackNotificationEventEnum, string> = {
  meetingEnd: 'Meeting End',
  meetingStart: 'Meeting Start',
  MEETING_STAGE_TIME_LIMIT_END: `Meeting ${MeetingLabels.TIME_LIMIT} Ended`,
  MEETING_STAGE_TIME_LIMIT_START: `Meeting ${MeetingLabels.TIME_LIMIT} Started`,
  TOPIC_SHARED: `Topic Shared`
}
```